src/api.h

```diff
@@ -877,6 +877,7 @@ void tic_core_synth_sound(tic_mem* tic);
 void tic_core_blit(tic_mem* tic);
 void tic_core_blit_ex(tic_mem* tic, tic_blit_callback clb);
 const tic_script_config* tic_core_script_config(tic_mem* memory);
+bool tic_core_enterp(tic_mem* tic, s32 hold, s32 period);
```

Comment:
Could you please move this function to the `studio.h`, because it's not related to the API.